client/app/account/billing/atInternetTracking.config.js

```diff
@@ -1,43 +1,52 @@
-angular.module("Billing.services").run(["$rootScope", "atInternet", "AUTORENEW_EVENT", "PAYMENT_EVENT", "OVH_ACCOUNT_EVENT", "FIDELITY_EVENT",
-    function ($rootScope, atInternet, AUTORENEW_EVENT, PAYMENT_EVENT, OVH_ACCOUNT_EVENT, FIDELITY_EVENT) {
-        "use strict";
-
-        $rootScope.$on(OVH_ACCOUNT_EVENT.CREDIT, trackClickEvent);
-        $rootScope.$on(OVH_ACCOUNT_EVENT.TRANSFER_TO_BANK_ACCOUNT, trackClickEvent);
-        $rootScope.$on(OVH_ACCOUNT_EVENT.ALERT, trackClickEvent);
-        $rootScope.$on(FIDELITY_EVENT.CREDIT, trackClickEvent);
-        $rootScope.$on(AUTORENEW_EVENT.DISABLE_AUTOMATIC_PAYMENT_FOR_DOMAINS, trackClickEvent);
-        $rootScope.$on(AUTORENEW_EVENT.TERMINATE, (event, data) => trackServiceClickEvent(event, data));
-        $rootScope.$on(AUTORENEW_EVENT.TERMINATE_AT_EXPIRATION, (event, data) => trackServiceClickEvent(event, data));
-        $rootScope.$on(AUTORENEW_EVENT.CANCEL_TERMINATE, (event, data) => trackServiceClickEvent(event, data));
-        $rootScope.$on(AUTORENEW_EVENT.ACTIVATE_AUTO_PAYMENT, (event, data) => trackServiceClickEvent(event, data));
-        $rootScope.$on(AUTORENEW_EVENT.DEACTIVATE_AUTO_PAYMENT, (event, data) => trackServiceClickEvent(event, data));
-        $rootScope.$on(AUTORENEW_EVENT.MODIFY, (event, data) => trackServiceClickEvent(event, data));
-        $rootScope.$on(AUTORENEW_EVENT.PAY, (event, data) => trackServiceClickEvent(event, data));
+angular.module('Billing.services').run(['$rootScope', 'atInternet', 'AUTORENEW_EVENT', 'PAYMENT_EVENT', 'OVH_ACCOUNT_EVENT', 'FIDELITY_EVENT',
+  function ($rootScope, atInternet, AUTORENEW_EVENT, PAYMENT_EVENT, OVH_ACCOUNT_EVENT,
+    FIDELITY_EVENT) {
```

Comment:
No alphabetical order ?

Reply:
According to the large amount of files updated in that pull request I didn't sort every parameters. I just let `ESLint` fix all errors encountered with the `--fix` flag.

Of course that would be a great enhancement to sort all parameters according to our guidelines but lets tackle this in smaller and separate tasks, right?

Reply:
No problem